traffic_ops/traffic_ops_golang/routes.go

```diff
@@ -393,11 +393,134 @@ func Routes(d ServerData) ([]Route, []RawRoute, http.Handler, error) {
+		{1.1, http.MethodGet, `deliveryservices/{id}/capacity/?(\.json)?$`, perlAPIHandler, 0, NoAuth, []Middleware{}},
+		{1.1, http.MethodGet, `deliveryservices/{id}/routing/?(\.json)?$`, perlAPIHandler, 0, NoAuth, []Middleware{}},
+		{1.1, http.MethodGet, `deliveryservices/{id}/state/?(\.json)?$`, perlAPIHandler, 0, NoAuth, []Middleware{}},
+		{1.1, http.MethodPost, `deliveryservices/request/?(\.json)?$`, perlAPIHandler, 0, NoAuth, []Middleware{}},
+		{1.1, http.MethodGet, `steering/{id}/targets/?(\.json)?$`, perlAPIHandler, 0, NoAuth, []Middleware{}},
+		{1.1, http.MethodGet, `steering/{id}/targets/{target_id}/?(\.json)?$`, perlAPIHandler, 0, NoAuth, []Middleware{}},
+		{1.1, http.MethodPost, `steering/{id}/targets/?(\.json)?$`, perlAPIHandler, 0, NoAuth, []Middleware{}},
+		{1.1, http.MethodPut, `steering/{id}/targets/{target_id}/?(\.json)?$`, perlAPIHandler, 0, NoAuth, []Middleware{}},
+		{1.1, http.MethodDelete, `steering/{id}/targets/{target_id}/?(\.json)?$`, perlAPIHandler, 0, NoAuth, []Middleware{}},
+		{1.1, http.MethodGet, `deliveryservices/xmlId/{xmlid}/sslkeys/?(\.json)?$`, perlAPIHandler, 0, NoAuth, []Middleware{}},
+		{1.1, http.MethodGet, `deliveryservices/hostname/{hostname}/sslkeys/?(\.json)?$`, perlAPIHandler, 0, NoAuth, []Middleware{}},
+		{1.1, http.MethodPost, `deliveryservices/sslkeys/add/?(\.json)?$`, perlAPIHandler, 0, NoAuth, []Middleware{}},
+		{1.1, http.MethodDelete, `divisions/name/{name}/?(\.json)?$`, perlAPIHandler, 0, NoAuth, []Middleware{}},
+		{1.1, http.MethodGet, `federations/?(\.json)?$`, perlAPIHandler, 0, NoAuth, []Middleware{}},
+		{1.1, http.MethodPost, `federations/?(\.json)?$`, perlAPIHandler, 0, NoAuth, []Middleware{}},
+		{1.1, http.MethodPut, `federations/?(\.json)?$`, perlAPIHandler, 0, NoAuth, []Middleware{}},
+		{1.1, http.MethodDelete, `federations/?(\.json)?$`, perlAPIHandler, 0, NoAuth, []Middleware{}},
+		{1.1, http.MethodGet, `federations/{fedId}/users/?(\.json)?$`, perlAPIHandler, 0, NoAuth, []Middleware{}},
+		{1.1, http.MethodPost, `federations/{fedId}/users/?(\.json)?$`, perlAPIHandler, 0, NoAuth, []Middleware{}},
+		{1.1, http.MethodDelete, `federations/{fedId}/users/{userId}/?(\.json)?$`, perlAPIHandler, 0, NoAuth, []Middleware{}},
+		{1.1, http.MethodGet, `federations/{fedId}/deliveryservices/?(\.json)?$`, perlAPIHandler, 0, NoAuth, []Middleware{}},
+		{1.1, http.MethodPost, `federations/{fedId}/deliveryservices/?(\.json)?$`, perlAPIHandler, 0, NoAuth, []Middleware{}},
+		{1.1, http.MethodDelete, `federations/{fedId}/deliveryservices/{dsId}/?(\.json)?$`, perlAPIHandler, 0, NoAuth, []Middleware{}},
+		{1.1, http.MethodGet, `federations/{fedId}/federation_resolvers/?(\.json)?$`, perlAPIHandler, 0, NoAuth, []Middleware{}},
+		{1.1, http.MethodPost, `federations/{fedId}/federation_resolvers/?(\.json)?$`, perlAPIHandler, 0, NoAuth, []Middleware{}},
+		{1.1, http.MethodPost, `federation_resolvers/?(\.json)?$`, perlAPIHandler, 0, NoAuth, []Middleware{}},
```

Comment:
can you add this?

{1.1, http.MethodGet, `federation_resolvers/?(\.json)?$`, perlAPIHandler, 0, NoAuth, []Middleware{}},